src/bin/commands/new-project/utils/imports.ts

```diff
@@ -8,33 +8,66 @@ interface Import {
 }
 
 export class Imports {
-  imports: Record<string, Import>;
+  private readonly imports: Record<string, Import>;
 
-  constructor(imports?: Record<string, Import>) {
-    this.imports = imports ?? {};
+  private constructor(imports: Record<string, Import>) {
+    this.imports = imports;
   }
 
-  addImport(lib: string, components: string[], type = false): void {
```

Comment:
Calls to `addImport` always had `type` set to `false`. This meant we could remove the `type` argument. Once that's done, the value of `addImport` quickly disappears, so we can just remove it!